gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/ActivityStack.groovy

```diff
@@ -0,0 +1,20 @@
+package com.novoda.gradle.command
+
+class ActivityStack extends AdbTask {
```

Comment:
Ah, sorry, did not read that carefully enough. 

The base class approach to provide adb access might be something to rethink in general.